index.html

```diff
@@ -559,11 +559,10 @@ <h3>Proofs</h3>
         </pre>
 
         <p>
-The example proof above suggests a ficticious
-`example-signature-2022` cryptography suite that produces a
-verifiable digital proof by presumptively transforming the input data using the
-JSON Canonicalization Scheme [[?RFC8785]] and then digitally signing it using an
-Ed25519 elliptic curve signature.
+The proof above uses the `jcs-eddsa-2022` cryptography suite [[?DI-EDDSA]] that
```

Comment:
I generally find things are better comprehended when the description (and any relevant warnings, etc.) precedes (rather than follows) the examples. This will take some minutes to adjust document-wide, along with changing all the `above` to `below`. I can do, within this PR, or as a standalone.....

Reply:
Agree, @TallTed, feel free to do that in this PR or one that layers on top of this one. Up to you, I'll take the changes either way.

Reply:
Nevermind, @TallTed, I went ahead an implemented your suggestion in https://github.com/w3c/vc-data-integrity/pull/105/commits/7575750ed5c24343e2e9152bcd86169956648dfe.